ci/azure/linux.yml

```diff
@@ -35,21 +35,24 @@ jobs:
         ci/incremental/install_miniconda.sh
         export PATH=$HOME/miniconda3/bin:$PATH
         echo "Setting up Conda environment"
+        source activate pandas-dev
```

Comment:
if I'm not wrong, the environment is created in the next file, and it doesn't exist here, so it can't be activated (and it's not needed)